docker/daemon/upload/deploy.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "net/http"
+
+import (
+	"io"
+	"log"
+	"os"
+)
+
+const (
+	BaseDir = "/var/naglfar/lib/"
+)
+
+func upload(w http.ResponseWriter, r *http.Request) {
+	fileName := r.Header.Get("fileName")
+	err := os.Remove(BaseDir + fileName)
+	if err != nil {
+		log.Println(err)
+	}
+	f, err := os.OpenFile(BaseDir+fileName, os.O_WRONLY|os.O_CREATE, 0600|0066)
+	if err != nil {
+		log.Println(err)
+	}
+	io.Copy(f, r.Body)
+	defer f.Close()
+	log.Println("upload successfully", fileName)
+}
+
+func main() {
```

Comment:
Put these code files into the cmd directory, please.

Reply:
the docker directory is only used to put Dockerfiles and some scripts.

Reply:
ok

Reply:
This part of the code has nothing to do with naglfar itself. It is the code of the internal image of docker and should not be placed under the cmd directory to create a new directory `external`?